src/data/firmware/1/releases.json

```diff
@@ -1,4 +1,18 @@
 [
+	{
+		"required": false,
+		"version": [1, 10, 0],
+		"bootloader_version": [1, 10, 0],
+		"min_bridge_version": [2, 0, 25],
+		"min_firmware_version": [1, 6, 2],
+		"min_bootloader_version": [1, 5, 0],
```

Comment:
I have copied & paste it from the latest release but is this correct? You need to install the intermediary FW first nope?